src/index.ts

```diff
@@ -1,13 +1,11 @@
 import axios from 'axios'
```

Comment:
will this get replaced in a separate pull request?

Reply:
And it's gone. Sorry about that.